ReactAndroid/devmainJava/build.gradle

```diff
@@ -75,8 +75,8 @@ dependencies {
     api fileTree(dir: "${annotationDir}", include: ['*.jar'])
     api 'com.android.support:appcompat-v7:26.0.2'
     api 'com.facebook.fbui.textlayoutbuilder:textlayoutbuilder:1.0.0'
-    api 'com.facebook.fresco:fresco:1.10.0'
```

Comment:
We no more build ReactNative in devmain and hence this file is not getting used now.